app/views/content/explore-teaching-advisers/_header.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render Campaign::HeaderComponent.new(
+  title: ["Explore", "teaching", "advisers"],
```

Comment:
this is something we can consider post go-live but I wonder if breaking the title onto three lines is the right thing to do - looks very blocky on mobile

![image](https://user-images.githubusercontent.com/87642394/190219686-1f44086c-6498-4133-8ff7-2020be4c7c89.png)

(might be more of a design consideration)